dynstringset.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2015 Michal Witkowski. All Rights Reserved.
+// See LICENSE for licensing terms.
+
+package flagz
+
+import (
+	"encoding/csv"
+	"fmt"
+	"strings"
+	"sync/atomic"
+	"unsafe"
+
+	flag "github.com/spf13/pflag"
+)
+
+// DynStringSet creates a `Flag` that represents `map[string]struct{}` which is safe to change dynamically at runtime.
+// Unlike `pflag.StringSlice`, consecutive sets don't append to the slice, but override it.
```

Comment:
Yes, and `DynStringSlice` also has an explicit comment to document that feature, so I presume it is worth documenting on `DynStringSet` too. Note that its being compared to `pflag.StringSlice`